test/native/test_lib/UnixStampTestNative.cpp

```diff
@@ -0,0 +1,112 @@
+#include <unity.h>
+#include <UnixStamp.hpp>
+#include <stdint.h>
+
+#define EXPECETED_CHERNOBYL_TIMESTAMP 514848225
+#define EXPECTED_ERA_END 951868799
+#define EXPECTED_ERA_START 951868800
+#define EXPECTED_EPOCH_LEFT_BORDER 0
+#define EXPECTED_EPOCH_RIGHT_BORDER 4294967295
+
+void setUp(void) {}
+
+void tearDown(void) {}
+
+civil_time fillTimeStruct(int16_t year, int8_t mon, int8_t day, int8_t hour, int8_t min, int8_t sec, int8_t tz)
+{
+    civil_time time;
+    time.year = year;
+    time.mon = mon;
+    time.day = day;
+    time.hour = hour;
+    time.min = min;
+    time.sec = sec;
+    time.tz = tz;
+    return time;
+}
+
+void verify_date_conversion_from_civil_to_epoch(uint32_t expected_unix, int16_t year, int8_t mon, int8_t day, int8_t hour, int8_t min, int8_t sec, int8_t tz)
+{
+    civil_time time = fillTimeStruct(year, mon, day, hour, min, sec, tz);
+    unixstamp unix = UnixStamp::convertTimeToUnix(time);
+    TEST_ASSERT_EQUAL(expected_unix, unix);
+}
```

Comment:
Why are we copying some code around? If this is reusable code, it should be extracted and reused. If the idea is to run all the tests on both the "native" (dev host) and the embedded platforms, it can be achieved without copying anything (we can discuss this in a chat).

Reply:
I tried to move common code(with unity tests) into separate files and store it test/common, but in this case VSCode demands to add this dir to include paths, and this code is compiled during _src_ compilation, which leads to compilation error, because unitty lib is not added for the main app.

Reply:
I added an issue: https://github.com/chifir/UnixStamp/issues/3